deepchem/__init__.py

```diff
@@ -5,6 +5,8 @@
 from __future__ import division
 from __future__ import unicode_literals
 
+__version__ = '2.0.dev0'
```

Comment:
@Agent007 I'd just make this 1.3.1 for simplicity. We're probably a ways from having a release candidate for 2.0 :)

Reply:
@rbharath Done! I'm looking for more tasks that I can help contribute to while building my deep learning expertise. Got any suggestions?